### PR TITLE
Set up development environment

### PR DESCRIPTION
**Pull request by hjheath**

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://109-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ Breaking changes result in a different major. UI changes that might break custom
 ### Changed
 - Internal: Removed `preact-router`.
 - Public: Removed URL routes for each step of the SDK flow.
-- Internal: Removed unused components - Dropdown and ActionBar
+- Internal: Removed unused components - Dropdown and ActionBar.
+- Internal: Use the staging backend when in development.
+# TODO: release and bump onfido-sdk-core version.
```
Comment (rfreitas) [inline CHANGELOG.md]: `#` creates a title on markdown, not a comment. comments are a bit noisy in markdown, see: http://stackoverflow.com/questions/4823468/comments-in-markdown

File: package.json [hunk 2]
```diff
@@ -3,12 +3,14 @@
   "version": "0.10.0",
   "description": "JavaScript SDK view layer for Onfido identity verification",
   "scripts": {
-    "predev": "npm run build",
+    "predev": "npm run build-dev",
```
Comment (rfreitas) [inline package.json]: won't this run two identical build steps when calling `npm run dev`?

File: src/index.html [hunk 3]
```diff
@@ -48,7 +48,14 @@
       }
 
       window.onload = function() {
-        var url = "https://gentle-gorge-17630.herokuapp.com/api"
+        const env = Onfido.env
```
Comment (rfreitas) [inline src/index.html]: I'm not sure if the webpack build script is transpiling javascript inside of an html. I would check that before using `const`.

Comment (hjheath) [inline src/index.html]: it's not transpiling, but there is widespread support of this syntax in browsers

**Comment by rfreitas** [comment]
Travis automatic deployment: https://109-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://109-pr-onfido-sdk-ui-onfido.surge.sh

File: src/index.html [hunk 4]
```diff
@@ -85,7 +85,8 @@
                 },
                 'face',
                 'complete'
-              ]
+              ],
+              url: 'wss://ws-staging.onfido.com'
```
Comment (rfreitas) [inline src/index.html]: maybe a more specific name

File: src/index.js [hunk 5]
```diff
@@ -25,13 +25,15 @@ const ContainerPure = ({ options, socket }) =>
 class Container extends Component {
   componentWillMount () {
     const { token } = this.props.options
```
Comment (rfreitas) [inline src/index.js]: const { token, url }

**Comment by rfreitas** [comment]
Travis automatic deployment: https://109-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://109-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://109-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://109-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://109-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**